frontend/src/features/seeds/routes/SeedDetails.tsx

```diff
@@ -0,0 +1,136 @@
+import { PlantsDto } from '../../../bindings/definitions';
+import PageLayout from '../../../components/Layout/PageLayout';
+import { findPlantById } from '../api/findPlantById';
+import { findSeedById } from '../api/findSeedById';
+import { SeedDto } from '@/bindings/definitions';
+import SimpleFormInput from '@/components/Form/SimpleFormInput';
+import PageTitle from '@/components/Header/PageTitle';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+('@/bindings/definitions');
+
+export function SeedDetails() {
+  const { id } = useParams();
+
+  const [seed, setSeed] = useState<SeedDto | null>(null);
+  const [plant, setPlant] = useState<PlantsDto | null>(null);
+
+  useEffect(() => {
+    // fetch seed
+    const _findOneSeed = async () => {
+      const seed = await findSeedById(Number(id));
+      setSeed(seed);
+
+      const plant = await findPlantById(Number(seed.plant_id));
+      setPlant(plant);
+    };
+    _findOneSeed();
+  }, []);
+
+  const currentYear = new Date().getFullYear();
+
+  return (
+    <PageLayout>
+      {seed && plant && (
+        <div>
+          <PageTitle title={seed?.name.toString()} />
+          <div className="mb-6 grid gap-8 md:grid-cols-2">
+            <SimpleFormInput
```

Comment:
Now fixed with `SimpleCard`. Not sure if Card is an appropriate name for such component, but any suggestions are welcome. I chose it because cards represent containers that group related content together and `Simple` suggests it is a simplified version of a proper card.